### ISSUE TITLE
Indicators for I3: (meta)data include qualified references to other (meta)data

### ISSUE THREAD

**Issue by makxdekkers** [RDA-FAIR/FAIR-data-maturity-model-WG#25]
![image](https://user-images.githubusercontent.com/7534601/60017571-53ac5680-9689-11e9-97aa-ebe025def421.png)

**Comment by makxdekkers** [comment]
Points raised in online meeting 3 on 18 June 2019

- The qualification cannot be tested logically. For that reason, it should be considered in the context of the discussion on ‘rich metadata’.
- Since vocabularies and terms change should this not have a temporal range for validity?
- General qualification is meant as unspecific links (like Web links). Contributor is already a specific qualification.

**Comment by RDA-FAIR** [comment]
![image](https://user-images.githubusercontent.com/47179213/60704314-5d914d80-9f04-11e9-8e7c-48d43ccbb577.png)

Here is a new version of the indicators!

**Comment by keithjeffery** [comment]
@Makx - could I3-02D include in 'specific' that it must be machine understandable (formal syntax, declared semantics)?
best
Keith

**Comment by makxdekkers** [comment]
Yes we can add that: Specific, machine-understandable qualification.

**Comment by RDA-FAIR** [comment]
Please find the current version of the indicator(s) and their respective maturity levels for this FAIR principle. Indicators and maturity levels will be presented, as they stand, to the next working group meeting for approval.  In the meantime, any comments are still welcomed.

The editorial team will now concentrate on weighing and prioritizing these indicators. More information soon.

![image](https://user-images.githubusercontent.com/47179213/62354687-bf49d500-b50d-11e9-8a71-8b768ed56d4d.png)

![image](https://user-images.githubusercontent.com/47179213/62354691-c244c580-b50d-11e9-88b6-293a3f5355ff.png)

**Comment by makxdekkers** [comment]
Peter Wittenburg at https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit?disco=AAAADadg-Vs

On I3-01D and I3-02D:
In case of big data (structured number sequences) there are no references to other data. Some formats such as for example JPEG, DICOM, netCDF, etc. support file headers which are structured by a schema and they may contain references. But in general this is not how people are working as far as I know. So the formulation is not approapriate.

**Comment by makxdekkers** [comment]
Peter, the formulation of the indicators is directly derived from the name of the principle. Are you suggesting that principle I3 is not appropriate for Big Data?

Please note that in the proposal for prioritisation of the indicators at https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/32, none of the indicators for I3 are mandatory -- so they are only to be evaluated if they are relevant for the resources under consideration. If they are not relevant for Big Data, they can be ignored.
Does that resolve your comment?

**Comment by keithjeffery** [comment]
@Peter-
are you suggesting that metadata about big data cannot/does not reference metadata about other digital assets?  In my experience big data metadata commonly references metadata describing other assets.

**Comment by RDA-FAIR** [comment]
On the 19/08/2019, Mark D Wilkinson commented

In the case of Big Data (structured number sequences), no references to other data will exist. In that case, the formulation is not appropriate.

> We could add "where possible".  ...but even in Big Data, there are often references to external concepts.  Those concepts should be captured using the GUID of that external concept, rather than just a "label".  ...of course, in many/most Big Data formats, it's too late to change their structure, so then it falls onto the Metadata to explain what each external reference in the data "blob" referrs-to.  Not a solved problem (at least, not in widespread use).  

> For small-data, the rule should apply, IMO.  For Big Data formats, we should encourage those who are inventing new formats to consider this FAIR rule when they invent their data structures.

Read his comment here: https://www.rd-alliance.org/group/fair-data-maturity-model-wg/post/discussion-items-fair-data-maturity-model-19-august-2019

**Comment by makxdekkers** [comment]
@markwilkinson @peterwittenburg @keithjeffery Please note that at https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/32#issue-479009587, none of the indicators for I3 are proposed to be mandatory, so they would be evaluated 'when relevant' -- so if Big Data does not have references to other data in the data, it would mean the data can still be FAIR.

**Comment by keithjeffery** [comment]
@makxdekkers  Makx; we need to 'put to bed' the Big Data discussion.  My position is clear:
(a) metadata concerning any digital object (including big data) should have qualified references as **mandatory** (since the reference to the big data digital object should exist in the metadata like it should for any other digital object);
(b) big data digital objects are recommended to have **qualified** references to other digital objects;

**Comment by makxdekkers** [comment]
@keithjeffery 
(a) I3 is about references to **other** (meta)data, not about the reference in metadata to the data it describes -- that is F3. I think it can't be mandatory that metadata must reference **other** data. It might be uncommon for data not to be related in any way to other data (i.e. previous research), but it could still be possible.
(b) So, you are proposing that I3-02D is Recommended rather than Optional

**Comment by keithjeffery** [comment]
@makxdekkers Makx - apologies, I always have problems distinguishing metadata and data (both being digital objects/assets).  However, is it not posible for metadata a describing digital object A to have references (1) to metadata b describing digital object B (i.e. the sense of I3) and also (2) to digital object B directly?  I do not see the coupling between metadata and digital object (i.e. a<->A and b<->B) as strictly as you imply.  I would expect a n:m (qualified) relationship between metadata digital objects and digital objects (a<->A, a<->B, b<->B, b<->A).
I would make I3-01M mandatory, I3-02M recommended; I3-01D optional, I3-02D optional.
Personally I would prefer any relationship between two digital objects which are not metadata to be through any associated (through qualified relationships) metadata referenging the digital objects since this would include attributes/properties/fields required for managing the interoperability (relevance, quality, trust, security, privacy, rights etc all of which are unlikely to be in a digital object in the form of a dataset).

**Comment by makxdekkers** [comment]
@keithjeffery 
I was not implying that a->B is not allowed. In fact, we have added indicators I3-02M and I3-04M (slide 23 in the presentation for the meeting on 12 September, https://www.rd-alliance.org/workshop-4) for a->B, i.e. metadata includes references to other data.
My argument was that a->A and b->B are under F3, while a->B and b->A are under I3. At least, that is how I read F3 and I3.
On making I3-01M mandatory, do you think that this can be required in all possible circumstances?

**Comment by keithjeffery** [comment]
@makxdekkers Makx: if qualified relationship metadata <-> digital object (i.e. a<->A) is covered under F3 then I3-01 can be recommended.

**Comment by bahimc** [comment]
Dear contributors,
 
Below you can find the indicators and their maturity levels in their current state as a result of the above discussions and workshops.
 
![image](https://user-images.githubusercontent.com/47179213/66299305-89511480-e8f3-11e9-8a5d-16851be6414c.png)

![image](https://user-images.githubusercontent.com/47179213/66299316-8c4c0500-e8f3-11e9-9f47-9b3a491b4cff.png)
 
**Please note that this thread is going to be closed**, within a short period of time.  The current state of the indicators, as of early October 2019, is now frozen, with the exception of the indicators for the principles that are concerned with ‘richness’ of metadata (F2 and R1). The current indicators will be used for the further steps of this WG, which are prioritisation and scoring.  Later on, they will be used in a testing phase where owners of evaluation approaches are going to be invited to compare their approaches (questionnaires, tools) against the indicators. The editorial team, in consultation with the Working Group, will define the best approach to test the indicators and evaluate their soundness. As such, the current set of indicators can be seen as an ‘alpha version’. In the first half of 2020, the indicators may be revised and improved, based on the results of the testing. **If you have any further comments, suggestions regarding that specific discussion, please share them with us. Besides, we invite you to have a look at the following two sets of issues.**
 
**Prioritisation**
 
[• Indicators prioritisation for Findability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/30)[• Indicators prioritisation for Accessibility](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/31)
[• Indicators prioritisation for Interoperability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/32)[• Indicators prioritisation for Reusability ](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/33)
 
**Scoring** 
 
[• Indicators for FAIRness | Scoring
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/34) 
We thank you for your valuable input!
